tests/integration/ceph_flex_test.go

```diff
@@ -87,9 +87,9 @@ func (s *CephFlexDriverSuite) SetupSuite() {
 	flexTestCluster := TestCluster{
 		namespace:               s.namespace,
 		storeType:               "bluestore",
-		storageClassName:        "",
+		storageClassName:        "gp2",
 		useHelm:                 false,
-		usePVC:                  false,
+		usePVC:                  true,
```

Comment:
This should also be from an env var, but by default `false`. Maybe just revert it for now since the focus is the smoke suite. Looks like the other tests also need to revert these as well.

Reply:
Done.